lib/manager/helm-values/readme.md

```diff
@@ -12,3 +12,5 @@ coreImage:
   repository: bitnami/harbor-core
   tag: 2.1.3-debian-10-r38
 ```
+
+If you need to change the versioning format, go to: INSERT PROPER INTERNAL LINK to `docs/usage/modules/versioning.md` HERE.
```

Comment:
Do we want to use a internal link or do we want to link to the full URL on the published documentation site?

I don't know how the internal link should look though, so please use a suggestion to get what you like best. 😄

Reply:
Also do you want to use wording like this maybe? It's your own suggestion, but just checking here. 😄 

> There is no fixed versioning convention for helm charts or the Docker images they reference, so at times the default SemVer versioning may not work. Please see X for information of why and how to customize versioning per-dependency.

Reply:
It's not semver that they use - it's a specific versioning we created for Docker's most common use case